types/types.go

```diff
@@ -223,4 +240,7 @@ type SystemContext struct {
 	// === docker.Transport overrides ===
 	DockerCertPath              string // If not "", a directory containing "cert.pem" and "key.pem" used when talking to a Docker Registry
 	DockerInsecureSkipTLSVerify bool   // Allow contacting docker registries over HTTP, or HTTPS with failed TLS verification. Note that this does not affect other TLS connections.
+	// if nil, the library tries to parse ~/.docker/config.json to retrieve credentials
+	// FIXME: the automatic credentials retrieval doesn't play with external stores also, see containers/image#111
```

Comment:
This is unrelated to this field AFAICS; if we are already tracking this as an issue, no need to add a FIXME here.